chain/indexer/Cargo.toml

```diff
@@ -8,6 +8,7 @@ edition = "2018"
 actix = "0.9"
 tracing = "0.1.13"
 futures = "0.3.5"
+openssl-probe = { git = "https://github.com/khorolets/openssl-probe.git", branch="feature/support-macports" }
```

Comment:
1. I think we should depend on a regular package and patch it (define a patch section).
2. I believe we should move the initialization of `openssl_probe` to `neard` (move from telemetry), so telemetry and genesis downloads work.